### ISSUE TITLE
Unable to see auth.log events in Dashboard

### ISSUE THREAD

**Issue by mitchellkingsley** [wazuh/wazuh#14559]
Hey!
I am fairly new to Wazuh, so please bear with me.
I currently have a `wazuh-monitor` set up with a  single `wazuh-agent`. They are configured as such:
-  agent: 
    - FreeBSD 13.1-Release
    -  v4.1.5
    - Installed through pkg
- monitor:
    - Debian 10
    - Installed through apt.

The issue that I have been having is that the `agent` seems to be able to connect to the `monitor` without any issues, and I am able to see certain alerts, such as 
```
Host-based anomaly detection event (rootcheck).
System Audit event
```

However, I am failing to see anything in the monitor from my `auth.log` logcollector rule. In my agent's `ossec.conf` I have the following stanza:
```
<localfile>
    <log_format>syslog</log_format>
    <location>/var/log/auth.log</location>
  </localfile>
```

and I can see the following log output on the agent's `ossec.log` whenever I make an authentication attempt:
```
2022/08/15 19:09:13 ossec-logcollector[71163] read_syslog.c:97 at read_syslog(): DEBUG: Reading syslog message: 'Aug 15 19:09:13 <authpriv.notice> test-agent sudo[71465]:   '...
2022/08/15 19:09:13 ossec-logcollector[71163] read_syslog.c:134 at read_syslog(): DEBUG: Read 1 lines from /var/log/auth.log
```

Looking at the `ossec.log` file on the monitor side, I don't see any movement or alerts when an authentication attempt is made, and it seems to me like the messages are not making it there. I have confirmed that a connection between the agent and monitor is made on service restart on the agent: 

```
2022/08/15 19:11:31 ossec-agentd: INFO: Trying to connect to server (REDACTED:1514/tcp).
2022/08/15 19:11:31 ossec-agentd: INFO: (4102): Connected to the server (REDACTED:1514/tcp).
```

Any guidance on what kind of steps I could take to get these logs to show up on the dashboard would be greatly appreciated, I am fairly stumped.

Thanks!

**Comment by pereyra-m** [comment]
Hi @mitchellkingsley !

If you were able to confirm that the agent is correctly reading the file, we need to check if the logs are arriving at the manager:

- First, run `/var/ossec/bin/agent_control -l` in the manager to make sure that the agent is `Active`
- Then, if any alert was generated you should find it in `/var/ossec/logs/alerts/alerts.log`, not in `ossec.log`
- If the events are being sent but not matching any rule, they will be discarded. You can temporarily enable the [logall](https://documentation.wazuh.com/4.3/user-manual/reference/ossec-conf/global.html#logall) tag in the manager to store **all** the events in `/var/ossec/logs/archives/archives.log`. You should see the log lines there.

Please share the results of these tests so we can find the cause of the problem.

Regards.

**Comment by mitchellkingsley** [comment]
Hey, recalling how this was resolved.

I believe the issue was that I was logging on FreeBSD. FreeBSD prepends a field in the logging output that isn't standard. As a result, Wazuh was grabbing the wrong fields and was unable to match against any rules (leading to no logging output). 

My solution was to create a service on the FreeBSD host that streamed the logs with the improper format into a new local log with the correct format, then I pushed that modified log to Wazuh which made everything work as expected. 

Thanks for the followup!
